charts/opentelemetry-collector/examples/daemonset-annotation-discovery/rendered/daemonset.yaml

```diff
@@ -0,0 +1,119 @@
+---
+# Source: opentelemetry-collector/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: example-opentelemetry-collector-agent
+  namespace: default
+  labels:
+    helm.sh/chart: opentelemetry-collector-0.128.1
+    app.kubernetes.io/name: opentelemetry-collector
+    app.kubernetes.io/instance: example
+    app.kubernetes.io/version: "0.128.0"
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/component: agent-collector
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: opentelemetry-collector
+      app.kubernetes.io/instance: example
+      component: agent-collector
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
```

Comment:
To avoid collecting Collector's own logs we should make sure that Collector Pods are properly annotated with `io.opentelemetry.discovery.logs/enabled: "false"`.